### PR TITLE
Added circular pattern

### LINKED ISSUE #29: Add circular pattern as feature
Required for webinar

### LINKED ISSUE #32: Add "My First Drawing" tutorial
Needed for 2nd webinar

### PR DESCRIPTION
**Pull request by skymakerjaime**
Closes #29 
Closes #32 

@axelSkyMaker said it was fine!
@kriffe let me know if it is enough for tomorrow's webinar.
@alexnil take a look too if you have time (complaints are forbidden)

### CODE REVIEW

**Comment by skymakerjaime** [comment]
New update! Take a look before merge

File: docs/circular-pattern.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+id: circular-pattern
+title: Circular Pattern
+sidebar_label: Circular Pattern
+---
+
+```typescript
+function getCircularPatternSketch(sketch: SKYSKETCH.Sketch, numberOfCopies: number, radialDistance: number, args?: {
```
Comment (kriffe) [inline docs/circular-pattern.md]: Perhaps change the name from "get..." to "generate...")?

File: docs/circular-pattern.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  const endAngle = args.endAngle !== undefined ? args.endAngle : 2 * Math.PI
+  const referenceAngle = args.referenceAngle !== undefined ? args.referenceAngle : 0
+  const addLast = args.addLast !== undefined ? args.addLast : false
+
+  const circularPatternSketch = new SKYSKETCH.Sketch()
+  const NC = addLast ? numberOfCopies + 1 : numberOfCopies
+  const angleRange = endAngle - startAngle
+
+  for (let i = 0; i < NC; i++) {
+    const patternSketch = sketch.clone()
```
Comment (kriffe) [inline docs/circular-pattern.md]: Rename variable sketch to sketchToUseInPattern

File: docs/circular-pattern.md (hunk 1 above)
Comment (skymakerjaime) [reply docs/circular-pattern.md]: Done

File: docs/circular-pattern.md (hunk 2 above)
Comment (skymakerjaime) [reply docs/circular-pattern.md]: Done

**Pull request merged**